### PR TITLE
Enterprise guides [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by mazameli**
Adds a guide for customer-facing analytics with full-app embedding as well as streamlines the general enterprise guide. Also adds new doc for full-app embedding, and consolidates the existing separate SAML and JWT SSO docs in the enterprise-guide into a single SSO doc.

The main things that would be good to review:
- The full-app-embedding doc
- The two new "overview" pages in each of the two guides
- TOC pages
- "Next" links at the end of each page in the two guides

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -13,10 +13,14 @@ This guide is for advanced users and those who will be setting up and maintainin
...
-## Enterprise Edition
+## Enterprise Edition Guides
+These guides cover things you can do with the [Enterprise edition of Metabase](https://metabase.com/enterprise/).
 
-#### [Enterprise Features Guide](enterprise-guide/start.md)
-Here's where to go for help using the features included in the Enterprise Edition of Metabase.
+#### [Providing analytics to your customers](./enterprise-customer-facing-guide/start.md)
+This guide will show you how you can embed Metabase to power analytics in your own web app, or set up your Metabase instance to allow customers to log in and view the data you want them to.
+
+#### [Managing large-scale deployments]((./enterprise-guide/start.md))
```
Comment (salsakran) [inline docs/README.md]: double parens here

File: docs/enterprise-customer-facing-guide/setting-up-sso.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+These settings allow Metabase to automatically get each user's email address and first and last name.
+
+The settings that Metabase defaults to here might work for you out of the box, but you can override them if you know that your settings are different.
+
+Each of these input boxes needs a URI that points to the location of a SAML attribute.
+
+
+
+
+## JWT-based Authentication
```
Comment (salsakran) [inline docs/enterprise-customer-facing-guide/setting-up-sso.md]: I think maybe the double `##` here isn't clear enough. It "feels" like it runs on from the SAML section vs being a different option

File: docs/enterprise-customer-facing-guide/full-app-embedding.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+
+### Choosing what to embed
+The exact next steps will differ depending on your specific needs and goals, but the basic tool you have at hand now is that you can make any link in your web app render a particular page from your Metabase instance.
+
+So if you have for example a "Stats" or "Analytics" page in your web app, you could have that page display one of your Metabase dashboards. What's powerful about this type of embedding vs. standalone chart or dashboard embeds though is that your users will be able to click on the individual charts in that dashboard to see them in more detail, and further explore them using drill-through, or even Metabase's graphical query builder.
+
+You can even display a specific Metabase collection in an embed to allow your users to browse through all the dashboards and questions that you've made available to them.
+
+### A note on drill-through and permissions
```
Comment (salsakran) [inline docs/enterprise-customer-facing-guide/full-app-embedding.md]: this should probably be at the top of this section

Comment (salsakran) [reply docs/enterprise-customer-facing-guide/full-app-embedding.md]: specifically - "when would I embed the whole app vs "normal" sharing/embedding

File: docs/enterprise-guide/setting-up-sso.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
+## Let you customers log in with SSO using SAML or JWT
+The Enterprise edition of Metabase lets you connect your SAML- or JWT-based SSO to it. Integrating your SSO with Metabase allows you to:
+
+* automatically pass user attributes from your SSO to Metabase in order to power data sandboxes
+* let your users access Metabase without re-authenticating.
+
+There are slightly different steps to take depending on whether your SSO solution uses SAML or JWT. We'll cover SAML first.
+
```
Comment (salsakran) [inline docs/enterprise-guide/setting-up-sso.md]: we should include a link to https://metabase.com/docs/latest/administration-guide/10-single-sign-on.html and note that these options exist.

**Review by salsakran — APPROVED** [review]
I think we'll need to tweak the nature of having duplicate information across two guides, but I do really like the core idea here - having a "getting started" style guide vs a user's manual. 

Good to go, though we'll tweak here and there as we go.

**Comment by mazameli** [comment]
Closing since we went another way with this.

**Pull request closed without merging**